recommendation/model/user_rec.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.preprocessing import OneHotEncoder
+from sklearn.metrics.pairwise import cosine_similarity
+
+# need a way to store similarity matrix and only update when posts are increased instead of redoing
+def get_matrix(users, n):
```

Comment:
I like your idea of relating users to each other if they like the same post. I think something we can consider for this algorithm is to give different users more weight. If user A and B like 8 same posts whereas user A and C like only 4 same posts, i think userB's other liked post should hold more weight to user A. Are there any scikit learn algorithms we can apply?

Please let me know if this is already considering this and I'm not fully understanding the code!